src/test/resources/Ds-algoFeatures/Array.feature

```diff
@@ -0,0 +1,408 @@
+#Author: your.email@your.domain.com
+#Keywords Summary :
+#Feature: List of scenarios.
+#Scenario: Business rule through list of steps with arguments.
+#Given: Some precondition step
+#When: Some key actions
+#Then: To observe outcomes or validation
+#And,But: To enumerate more Given,When,Then steps
+#Scenario Outline: List of steps for data-driven as an Examples and <placeholder>
+#Examples: Container for s table
+#Background: List of steps run before each of the scenarios
+#""" (Doc Strings)
+#| (Data Tables)
+#@ (Tags/Labels):To group Scenarios
+#<> (placeholder)
+## (Comments)
+#Sample Feature Definition Template
+@tag 
+Feature: Array
+  I want to use this template for my feature file
+
+  @tag1
+  Background: The user is logged in to DS Algo portal
+  Scenario: The user is logged in to DS Algo portal
+    Given The user is on Signin page of DS Algo portal
+    When The user enter valid username "Numpysdet108" and password "VP108AK@08"
+    Then The user redirected to homepage
+   
+
+  @tag2 Home
+  Background: The user is logged in to DS Algo portal
+  Scenario Outline: The user is able to navigate to "ARRAY" Data Structure Page
+    Given The user is on the "home page" after logged in
+    When The user clicked on Array "Get Started" Button
+    Then The user redirected to "ARRAY" Data Structure Page
+
+
+	@tag3 Array
+  Background: The user is logged in to DS Algo portal
+  Scenario Outline: The user is able to navigate to "ARRAY" Data Structure Page
+    Given The user is on the "Array page" after logged in
+    When The user clicks "Arrays in Python" button
+    Then The user redirected to "Arrays in Python" Page
+    
+  @tag4 Array
+  Background: The user is logged in to DS Algo portal
+  Scenario Outline: The user is able to navigate to a page having an tryEditor from Arrays in Python page
+    Given The user is on the "Arrays in Python page" after logged in
+    When The user clicks "Try Here" button
+    Then The user should be redirected to a page having an tryEditor with a Run button to test
+    
+  @tag5 Array
+  Background: The user is logged in to DS Algo portal
+  Scenario Outline: The user is able run code in tryEditor of Arrays in Python page
+    Given The user is in a page having an tryEditor with a Run button to test
+    When "The user enter valid python code in tryEditor from sheet "print "Welcome to Ds-Algo"" and "<1>"
+		And The user clicks on run button
+		Then "The user should be presented with Run result"
+		
+		
+	@tag6 Array
```

Comment:
why tag6 doesn't show in red indentation ?.
Have you tested this feature in local env ?..

Reply:
i did some changes in Array feature file